### PR TITLE
Add Travis CI integration

### LINKED ISSUE #9: Add integration with Travis CI
Please configure integration with TravisCI:
* [x] [Enable repo testing in TravisCI](https://travis-ci.com/kpi-web-guild/django-girls-blog-schenkh)
* [x] Add TravisCI badge (master branch) to `README.md` with a single commit. (click Build Status next to octocat icon on [this page](https://travis-ci.com/kpi-web-guild/django-girls-blog-schenkh))
* [x] Create new branch called `feature/travis-ci-integration` and do all the following steps within that branch.
* [x] Add `.travis.yml` using `pre-commit` as a script. Set it up to run Python 3.7 env and add PIP caching. Ensure you install `pre-commit` before running tests:
```yaml
language: python
python: 3.7
cache: pip
before_install: python -m pip install -U pre-commit
script: python -m pre_commit run --all-files
```
* [x] Push the branch and create a Pull Request for it.

### PR DESCRIPTION
**Pull request by schenkh**
Resolves #9

### CODE REVIEW

**Comment by webknjaz** [comment]
> Add .travis.yml

It's an obvious implementation detail. You should describe the high-level effect of this commit.
Like "Configure Travis CI" or "Add initial Travis CI config file"

**Comment by webknjaz** [comment]
> Add integration with Travis CI in feature/travis-ci-integration

This PR description is stating too many implementation details. The branch name is visible in the UI and in general, is not useful for the reader.
Also, since it doesn't add value to the PR title you may as well just drop it and only keep `Resolves ...` line there.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: python
```
Comment (webknjaz) [inline .travis.yml:2]:
```suggestion
dist: xenial
language: python
```

**Review by webknjaz — CHANGES_REQUESTED** [review]
Interesting. Your Travis CI uses Ubuntu Trusty VM which doesn't support Python 3.7.
Instruct it to use Xenial instead explicitly.

In general, for this PR to be acceptable, make sure that CI builds are green.

File: .travis.yml (hunk 1 above)
Comment (webknjaz) [reply .travis.yml:2]: @schenkh you could've just hit "Commit suggestion" right here in the UI ^

**Review by webknjaz — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (schenkh) [reply .travis.yml:2]: Ah! Ok, thanks

**Comment by webknjaz** [comment]
Oh, you'll need to wait for pre-commit config to be checked in first. (See the log)

**Comment by webknjaz** [comment]
@schenkh don't merge PRs having red CI.

**Comment by schenkh** [comment]
Ok, I thought it fails because this branch is not merged.

**Comment by webknjaz** [comment]
No guesses, go to https://github.com/kpi-web-guild/django-girls-blog-schenkh/pull/16/checks and open builds to see what's happening there. Also, this branch was out-of-date. Rebasing it would've updated CI statuses with more relevant info.

**Pull request merged**